server/repository/src/migrations/v2_03_00/mod.rs

```diff
@@ -17,12 +17,12 @@ impl Migration for V2_03_00 {
 #[cfg(test)]
 #[actix_rt::test]
 async fn migration_2_03_00() {
-    use v2_02_01::V2_02_01;
+    use v2_02_02::V2_02_02;
 
     use crate::migrations::*;
     use crate::test_db::*;
 
-    let previous_version = V2_02_01.version();
+    let previous_version = V2_02_02.version();
```

Comment:
Not sure why 2_03_00 is in main but it is...

Reply:
Yea odd! I might be mistaken, but doesn't that mean some sites might be on 2_03 as far as migrations are concerned, or is it checked against the package.json version somewhere? If not won't your new migrations be missed who are already on 2.2 (which is secretly 2_3 migration) ?

Reply:
It's commented out from the migrations list, so pretty sure no one has run it...